src/install-opentofu.ps1

```diff
@@ -290,8 +373,19 @@ function installStandalone() {
     $dlFiles += $sumsFile
     if (!$skipVerify)
     {
-        $dlFiles += $sigFile
-        $dlFiles += $certFile
+        if ($verifyMethod -eq "cosign")
+        {
+            $dlFiles += $sigFile
+            $dlFiles += $certFile
+        } elseif ($verifyMethod -eq "gpg") {
+            if (@("1.6.0-alpha1","1.6.0-alpha2","1.6.0-alpha3","1.6.0-alpha4","1.6.0-alpha5","1.6.0-beta1","1.6.0-beta2","1.6.0-beta3","1.6.0-beta4","1.6.0-beta5","1.6.0-rc1").Contains($opentofuVersion)) {
+                throw [InstallRequirementNotMetException]::new("${bold}Additional tools are needed for the installation. Please read the following text carefully!${normal}`n`nOpenTofu version ${opentofuVersion} is not GPG-signed. This installer tries to verify that the OpenTofu version downloaded has been ${bold}signed by OpenTofu and has not been tampered with${normal}. This is only possible if either cosign or GPG is installed on the system. You have GPG installed, but only OpenTofu 1.6.0 and later are GPG-signed. You have the following options:`n`n1. ${bold}Install cosign${normal} and add it to your ${magenta}PATH${normal} or provide the ${magenta}-cosignPath${normal} parameter to your cosign installation.`n2. ${bold}Disable integrity verification${normal} with ${magenta}-skipVerify${normal} (${red}not recommended${normal}).")
```

Comment:
It might be pretty obvious, but maybe we also want to suggest to the user to install openTofu > 1.6.0 as one of the options?

Reply:
I was hoping to get rid of this code eventually. Not sure if anyone is using it.

Reply:
I wouldn't be against nuking this logic now. And saying we only support 1.6.0 stable and above with this script.

Reply:
This is in both scripts, let's handle that in a separate PR.